content/en/integrations/guide/azure-programmatic-management.md

```diff
@@ -133,21 +131,21 @@ See the [Virtual Machine Extension resource][10] in the Terraform registry for m
 {{< partial name="whats-next/whats-next.html" >}}
 
 
+[1]: https://docs.datadoghq.com/integrations/guide/azure-portal/
 [2]: https://learn.microsoft.com/en-us/cli/azure/datadog?view=azure-cli-latest
 [3]: /getting_started/site/
-[4]: /integrations/guide/azure-portal/
+[4]: /integrations/guide/azure-manual-setup/?tab=azurecli#integrating-through-the-azure-cli
 [5]: /integrations/azure/
-[6]: /agent/basic_agent_usage/ansible/
-[7]: /integrations/azure_container_service/
+[6]: /integrations/guide/azure-manual-setup/?tab=azurecli#integrating-through-the-azure-portal
 [9]: https://developer.hashicorp.com/terraform/language/providers/configuration
 [10]: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/virtual_machine_extension
 [11]: https://app.datadoghq.com/account/settings/agent/latest
 [12]: /agent/guide/agent-configuration-files/?tab=agentv6v7
 [13]: https://www.terraform.io
 [14]: https://learn.microsoft.com/en-us/azure/partner-solutions/datadog/overview
 [15]: https://registry.terraform.io/providers/DataDog/datadog/latest/docs
-[16]: https://learn.microsoft.com/en-us/cli/azure/monitor/diagnostic-settings?view=azure-cli-latest#az-monitor-diagnostic-settings-create
 [17]: https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/integration_azure
 [18]: /logs/guide/azure-logging-guide
 [19]: https://app.datadoghq.com/monitors/recommended
 [20]: /monitors/notify/#notify-your-team
+[21]: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/guides/service_principal_client_secret#creating-a-service-principal
```

Comment:
Do you intend on adding this link to a section above?

Reply:
Thanks for flagging, removed!